src/routes/product.route.ts

```diff
@@ -14,4 +15,7 @@ router.patch(
   protectRoute,restrictTo(userRole.seller),
   updateProductAvailability
 );
+
+router.delete('/deleteProduct/:product_id',protectRoute,restrictTo(userRole.seller),deleteProduct)
```

Comment:
The middleware can also check that its not just a seller role but the right seller